pyproject.toml

```diff
@@ -12,9 +12,13 @@ requests = "^2.31.0"
 pytest = "^8.1.1"
 selenium = "^4.18.1"
 gunicorn = "^22.0.0"
+pymongo = "^4.10.1"
 
 [tool.poetry.dev-dependencies]
 
+[tool.poetry.group.dev.dependencies]
+mongomock = "^4.2.0.post1"
```

Comment:
What an odd version string! Not important, but I [learned something looking this up](https://peps.python.org/pep-0440/#post-releases)
> Some projects use post-releases to address minor errors in a final release that do not affect the distributed software (for example, correcting an error in the release notes).

Reply:
That's interesting, thanks for flagging!